### PR TITLE
merge with sk-master

### PR DESCRIPTION
**Pull request by askarovit**

### CODE REVIEW

File: logic/transaction.js [hunk 1]
```diff
@@ -1128,6 +1128,13 @@ Transaction.prototype.afterSave = async function (trs, cb) {
     if (!tx_type) {
         return setImmediate(cb, 'Unknown transaction type ' + trs.type);
     } else {
+        const lastTransaction = await self.scope.db.one(sql.getTransactionById, { id: trs.id});
+        await utils.addDocument({
+            index: 'trs',
+            type: 'trs',
+            body: lastTransaction,
+            id: lastTransaction.rowId
```
Comment (Verbalist) [inline logic/transaction.js]: lastTransaction
```suggestion
            id: lastTransaction.Id
```

File: logic/sendFreezeOrder.js [hunk 2]
```diff
@@ -168,7 +168,8 @@ SendFreezeOrder.prototype.apply = async function (trs, block, sender, cb) {
 		await utils.updateDocument({
             index: 'stake_orders',
             type: 'stake_orders',
-            body: stakeOrders.prev
+            body: stakeOrders.prev,
+			id: stakeOrders.prev.stakeId
```
Comment (Verbalist) [inline logic/sendFreezeOrder.js]: stakeOrders.prev.stakeId
```suggestion
stakeOrders.prev.id
```

File: modules/transport.js [hunk 3]
```diff
@@ -513,6 +515,16 @@ Transport.prototype.onNewBlock = function (block, broadcast) {
...
 			library.network.io.sockets.emit('blocks/change', block);
 		});
+
+		library.db.one(sqlBlock.getBlockByHeight, { height: block.height })
+			.then(function (lastBlock) {
+				utils.addDocument({
+					index: 'blocks_list',
+					type: 'blocks_list',
+					body: lastBlock,
+					id: lastBlock.b_height
```
Comment (Verbalist) [inline modules/transport.js]: same

**Comment by Verbalist** [comment]
Change in old place from stakeId to id and in new place use only id

File: logic/sendFreezeOrder.js (hunk 2 above)
Comment (askarovit) [reply logic/sendFreezeOrder.js]: I think is better use "stakeId" because it field is table's primary key.

**Pull request merged**